app/src/main/kotlin/com/wire/android/ui/debug/DebugDataOptions.kt

```diff
@@ -512,6 +512,6 @@ fun PreviewOtherDebugOptions() {
         handleE2EIEnrollmentResult = {},
         dismissCertificateDialog = {},
         checkCrlRevocationList = {},
-        dependenciesMap = persistentMapOf()
+        dependenciesMap = persistentMapOf("avs" to "1.0.0", "core-crypto" to "1.0.0")
```

Comment:
maybe we can have a default value of null just in case something is wrong it will not show 1.0.0 which is incorrect value

```suggestion
        dependenciesMap = persistentMapOf("avs" to "null", "core-crypto" to "null")
```

Reply:
These values are only for a preview 😄 
I changed these names for some more generic ones to avoid confusion

Reply:
ohhhhh my bad, I saw in the pr discretion mentioning empty state and thought it was changed here